### PR TITLE
overlay/live-generator: Avoid overlayfs on squashfs in RHEL8

### PR DESCRIPTION
**Pull request by cgwalters**
It looks like the `squashfs` in RHEL8 is too old to support
being used as a "lowerdir" for `overlayfs`, which is too bad
because the two go together really well.  Hack around this
by copying `/etc` instead.

### CODE REVIEW

File: overlay.d/05core/usr/lib/dracut/modules.d/20live/live-generator [hunk 1]
```diff
@@ -119,14 +119,43 @@ Before=ignition-mount.service
...
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/bin/cp -a /sysroot/etc /writable/etc-copy
+EOF
+
+    common_etcvar_unit > "${UNIT_DIR}/sysroot-etc.mount"
+cat >>"${UNIT_DIR}/sysroot-etc.mount" <<EOF
+After=sysroot-etc-copy.service
+Requires=sysroot-etc-copy.service
+Before=ignition-ostree-populate-var.service
```
Comment (bgilbert) [inline overlay.d/05core/usr/lib/dracut/modules.d/20live/live-generator]: Why is this needed here but not in the other branch?

Comment (cgwalters) [reply overlay.d/05core/usr/lib/dracut/modules.d/20live/live-generator]: I think this is leftover cruft from when I was confused and thought we were doing an overlayfs for `/var` too.  Testing removing it...

Comment (cgwalters) [reply overlay.d/05core/usr/lib/dracut/modules.d/20live/live-generator]: OK &lt;some distractions later&gt; yep removing that works fine!
